### ISSUE TITLE
Provide a non-interactive prop

### ISSUE THREAD

**Issue by lorensr** [ikr/react-star-rating-input#24]
`<StarRatingInput interactive={false} />` 

Say User A inputs a rating of 3 stars using this input. It would be nice to be able to display 3 stars as User A's rating to User B, and not allow User B to change the stars shown on hover.

**Comment by ikr** [comment]
Yeah, I see that being useful. Don't know when I'll be able to get to it though. Will gladly review an MR for the feature, if it's covered with tests.
